exp/etcdrestore/controllers/etcdsnapshotrestore_controller_test.go

```diff
@@ -0,0 +1,101 @@
+/*
+Copyright © 2023 - 2024 SUSE LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package controllers
+
+import (
+	. "github.com/onsi/ginkgo/v2"
+	// . "github.com/onsi/gomega"
+	bootstrapv1 "github.com/rancher/cluster-api-provider-rke2/bootstrap/api/v1beta1"
+	// corev1 "k8s.io/api/core/v1"
+)
+
+var (
+	rke2Config              *bootstrapv1.RKE2Config
+	serviceAccountName      string
+	serviceAccountNamespace string
+	planSecretName          string
+	serverUrl               string
+	pem                     string
+	systemAgentVersion      string
+	token                   []byte
+)
+
+var _ = Describe("RKE2ConfigWebhook tests", func() {
+	// It("Should create a role with the correct properties", func() {
```

Comment:
should the code below removed if commented out?

Reply:
This was committed by accident, will remove it. Thanks